### PR TITLE
refactor(console): replace mfa upsell notification with `InlineUpsell` component

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Replace mfa upsell notification with `InlineUpsell` component

### Updates
- Support `actionButtonText` prop for the `InlineUpsell` component.
- Replace the translation text link component (`a`) with the `ContactUsPhraseLink` component. 
- Remove all `target=_blank` props for all email link, since it will cause the page to flash when the link is clicked.

## Testing

image

## Checklist

~~- [ ] `.changeset`~~
~~- [ ] unit tests~~
~~- [ ] integration tests~~
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/components/InlineUpsell/index.tsx [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
 import useTenantPathname from '@/hooks/use-tenant-pathname';
 
+import ContactUsPhraseLink from '../ContactUsPhraseLink';
+
 import * as styles from './index.module.scss';
 
 type Props = {
   className?: string;
   for: TFuncKey<'translation', 'admin_console.upsell.paywall'>;
+  actionText?: AdminConsoleKey;
```
Comment (gao-sun) [inline packages/console/src/components/InlineUpsell/index.tsx:16]: better to add TSDoc comment for this prop. should be clear and show the default value

Comment (xiaoyijun) [reply packages/console/src/components/InlineUpsell/index.tsx:19]: Updated.
image

Comment (gao-sun) [reply packages/console/src/components/InlineUpsell/index.tsx:19]: "Optional {@link AdminConsoleKey} for the action text" exactly equals to `actionText?: AdminConsoleKey`?

Comment (gao-sun) [reply packages/console/src/components/InlineUpsell/index.tsx:19]: it's more about what is action text? e.g. "The text to be shown for ..."

Comment (gao-sun) [reply packages/console/src/components/InlineUpsell/index.tsx:19]: hey bro you can google for the syntax

Comment (xiaoyijun) [reply packages/console/src/components/InlineUpsell/index.tsx]: Updated.

**Review by gao-sun — APPROVED** [review]

**Pull request merged**